support/troubleshooting/investigating-pod-issues.adoc

```diff
@@ -1,30 +1,30 @@
 [id="investigating-pod-issues"]
-= Investigating Pod issues
+= Investigating pod issues
 include::modules/common-attributes.adoc[]
 :context: investigating-pod-issues
 
 toc::[]
 
-{product-title} leverages the Kubernetes concept of a Pod, which is one or more containers deployed together on one host. A Pod is the smallest compute unit that can be defined, deployed, and managed on {product-title} {product-version}.
+{product-title} leverages the Kubernetes concept of a pod, which is one or more containers deployed together on one host. A pod is the smallest compute unit that can be defined, deployed, and managed on {product-title} {product-version}.
```

Comment:
Sort of. From upstream:

> A Pod (as in a pod of whales or pea pod) is a group of one or more containers, with shared storage/network resources, and a specification for how to run the containers. A Pod's contents are always co-located and co-scheduled, and run in a shared context. A Pod models an application-specific "logical host": it contains one or more application containers which are relatively tightly coupled. In non-cloud contexts, applications executed on the same physical or virtual machine are analogous to cloud applications executed on the same logical host.

> The shared context of a Pod is a set of Linux namespaces, cgroups, and potentially other facets of isolation - the same things that isolate a Docker container. Within a Pod's context, the individual applications may have further sub-isolations applied.

So you can have containers deployed together on a host (by an accident of k8s resource scheduling), but they aren't necessarily a _pod_.

What makes a pod is the specific isolation context shared amongst containers within it.

Maybe _deployed together_ is close enough, and I am being pedantic, but if I knew nothing about Kubernetes, I might get the wrong idea of what a pod is.

Maybe that's outside the scope of this update, though.